README.md

```diff
@@ -34,7 +36,10 @@ Currently, seven `basic` types of data are supported:
     3. an url
 - `ImageLabelManifest`: encapsulates one single image-level annotation
 - `CategoryManifest`: encapsulates the information about a category, such as its name and super category, if applicable
-- `VisionDataset` is an iterable dataset class that consumes the information from `DatasetManifest`.
+- `MultiImageLabelManifest`: encapsulates one multi-image annotation, it contains the referred image indices and labels.
```

Comment:
encapsulates one annotation with multiple images and one text involved.

Can label be any type or key_value_pair only?

Reply:
It is an abstract class, user is expected to write their own class (e.g. KeyValuePairLabelManifest)